docs/tables/azure_storage_container.md

```diff
@@ -0,0 +1,31 @@
+# Table: azure_storage_container
+
+A container organizes a set of blobs, similar to a directory in a file system. A storage account can include an unlimited number of containers, and a container can store an unlimited number of blobs.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  type,
+  account_name
+from
+  azure_storage_container;
+```
+
+### List containers which are not publicly accessible
+
+```sql
+select
+  name,
+  id,
+  type,
+  account_name
+from
+  azure_storage_container
+where
+  public_access = 'None';
+```
```

Comment:
Add more example 

1. List the containers which are either leased or lease broken state. If leased pls check if the lease_duration is infinite.
2. Check the retention days ending in next 7 days
3. List for which the legal_hold is true.

Reply:
doc updated